internal/pkg/service/appsproxy/dataapps/api/config.go

```diff
@@ -27,6 +28,7 @@ type AppID string
 type AppConfig struct {
 	ID             AppID              `json:"-"`
 	Name           string             `json:"name"`
+	ProjectID      string             `json:"projectId"`
```

Comment:
if you are adding projectID, can you fix a json tag for Name? json:"appName"

Reply:
Of course, I didn't known there is a typo/bug.

Reply:
See:
- https://github.com/keboola/keboola-as-code/pull/1709/commits/f31ca7ec4114189041421a5b22e8304903757fb2
- https://github.com/keboola/keboola-as-code/pull/1709/commits/9729da67ac76b10cbf9d375a7e2dec2515c1866d